tools/evalkit/tests/test_stats.py

```
@@ -0,0 +1,159 @@
+"""evalkit.stats 测试：差分对照（独立推导的已知数学值）+ hypothesis 属性 + 边界。
+
+参照值独立推导、不复用被测代码：泊松 rule-of-three 与 Garwood chi2 表值；CP 定义
+方程（math.comb 直算 CDF）；Wilson 闭式手算锚定；kappa 教科书 2x2（κ=0.4）。
+"""
+
+import math
+
+import pytest
+from evalkit.stats import *
```

Ruff (0.16.2)

[error] `from evalkit.stats import *` used; unable to detect undefined names

(F403)